src/redux/books/books.js

```diff
@@ -1,17 +1,60 @@
+import axios from 'axios';
+
 const ADD_BOOK = 'bookStore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+const GET_BOOKS = 'bookStore/books/GET_BOOKS';
 
 const initialState = [];
 
-export const addBook = (payload) => ({
-  type: ADD_BOOK,
-  payload,
-});
+export const addBook = (payload) => async (dispatchBook) => {
+  const books = {
+    title: `${payload.title} by ${payload.author}`,
+    category: `${payload.category}`,
+    item_id: `${payload.id}`,
+  };
+  await axios.post('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/mHtXvOlvqbaSoEZBEt9h/books', books);
+  dispatchBook({
+    type: ADD_BOOK,
+    payload,
+  });
+};
+
+export const getBooks = () => async (get) => {
```

Comment:
- [ ] Your GET action should return an array of objects, which will look similar to the array given below. Currently, it is missing `category`.

```
{
    "item1": [
        {
            "title": "The Handmaid's Tale",
            "category": "Fiction"
        }
    ],
    "item2": [
        {
            "title": "Great Expectations",
            "category": "Classics"
        }
    ]
}
```